src/main/java/org/broadinstitute/hellbender/tools/walkers/gnarlyGenotyper/GnarlyGenotyperEngine.java

```diff
@@ -88,27 +88,34 @@ public VariantContext finalizeGenotype(final VariantContext variant, final Varia
         //GenomicsDB or Evoquer merged all the annotations, but we still need to finalize MQ and QD annotations
         //return a VC with the finalized annotations and dbBuilder gets the raw annotations for the database
 
+        final List<Allele> targetAlleles;
```

Comment:
I haven't looked closely at these GnarlyGenotypeEngine changes but are they necessary for this branch? Could they be separated from the ValidateVariants changes?